scripts/delay_filter_run.py

```diff
@@ -45,7 +45,7 @@
     cache_dir=ap.cache_dir, res_outfilename=ap.res_outfilename,
     clobber=ap.clobber, write_cache=ap.write_cache, external_flags=ap.external_flags,
     read_cache=ap.read_cache, mode=ap.mode, overwrite_flags=ap.overwrite_flags,
-    factorize_flags=ap.factorize_flags, time_thresh=ap.time_thresh,
+    factorize_flags=ap.factorize_flags, apply_flag_to_nsample=ap.apply_flag_to_nsample, time_thresh=ap.time_thresh,
```

Comment:
I don't think you actually added this to the argparser

Reply:
Thanks! Should be fixed now!